metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/objectImporters/ObjectsImporter.java

```diff
@@ -0,0 +1,38 @@
+/*
+ *
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package uk.ac.ebi.ampt2d.metadata.importer.objectImporters;
+
+import uk.ac.ebi.ampt2d.metadata.persistence.entities.Analysis;
+import uk.ac.ebi.ampt2d.metadata.persistence.entities.ReferenceSequence;
+import uk.ac.ebi.ampt2d.metadata.persistence.entities.Sample;
+import uk.ac.ebi.ampt2d.metadata.persistence.entities.Study;
+
+import java.util.List;
+import java.util.Set;
+
+public interface ObjectsImporter {
+
+    List<Study> importStudy(Set<String> accessions);
```

Comment:
Since ObjectsImporter is no more visitor, can it be 
public interface ObjectsImporter<Obj> {
List<Obj> import(Set<String> accessions); }
Each class will have its own implementation; in each package has separate implementation of objects